stories/Button.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Button from 'ui/components/Button';
+
+const label = 'Hello Button';
+
+storiesOf('Button', module)
+	.add('default (no buttonType)', () => <Button>{label}</Button>)
```

Comment:
I left out href and top and noLink props as they didn't seem to have an effect on the look. Let me know if you think these still should be added.